runflake8

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+flake8 --max-line-length=110 \
+       --exclude=./docs/,./build/,./setup.py,./django_netjsongraph/migrations/ || exit 1
```

Comment:
See https://github.com/openwisp/django-netjsonconfig/blob/master/runflake8#L4 for an example of how to exclude test settings files